scripts/lib/config.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
```

Comment:
I don't think that you need any of this stuff. We'll just publish this to npm using the built-in tools

Reply:
So get rid of cpplint?